content/en/real_user_monitoring/browser/collecting_browser_errors.md

```diff
@@ -140,6 +141,86 @@ try {
 {{% /tab %}}
 {{< /tabs >}}
 
+### Collect React rendering errors
+
+You can instrument the React error boundaries to monitor React rendering errors using the RUM Browser SDK `addError()` API.
+
+The collected rendering error stack contains the component stack, which is unminified like any other error stack traces once you've uploaded sourcemaps.
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏The collected rendering errors contain a component stack, which is unminified like any other error stack traces once you've uploaded sourcemaps.